.gitignore

```diff
@@ -14,7 +14,6 @@ compile_commands.json
 .cache
 **/build/*
 build/*
-build*
```

Comment:
I would keep this.
On project with 2 MCUs I often have `build_nrf91` and `build_nrf52` folders, for example.

Reply:
This is just for this specific project, irnas-zephyr-template still has it.